actions/signed-commits/actions/signed-commits/dist/actions/signed-commits/package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Delete.

This happens because you have to build and run tests from the root dir of the repo, because there's no way to set CWD and relative paths in our nx commands.